docs/userGuide/syntax/thumbnails.mbdf

```diff
@@ -0,0 +1,46 @@
+## Thumbnails
+
+**A `thumbnail` component allows you to insert thumbnails using text, images, or icons.**
+
+<include src="outputBox.md" boilerplate >
+<span id="code">
+
+```html
+<thumbnail circle src="{{baseUrl}}/images/deer.jpg" size="150"/>
+<thumbnail circle text=":book:" background="#333" fontcolor="white" size="100"/>
+<thumbnail circle text="___CS___" background="#ccf" fontcolor="#339" border="3px solid #339" size="100"/>
+<thumbnail text=":fas-book:" fontcolor="darkgreen" size="100"/>
+```
+</span>
+<span id="output">
+<thumbnail circle src="{{baseUrl}}/images/deer.jpg" size="150"/>
+<thumbnail circle text=":book:" background="#333" fontcolor="white" size="100"/>
```

Comment:
Use kebab case for tag attributes `font-color` & `font-size`. Since I mentioned in the `vue-strap` repo to switch to camelCase prop names.

Also, does font color white change anything for the second example? Since the text is just a book emoji.